great_expectations/cli/toolkit.py

```diff
@@ -28,6 +30,9 @@
 from great_expectations.datasource import BaseDatasource
 from great_expectations.validator.validator import Validator
 
+if TYPE_CHECKING:
+    from great_expectations.datasource import LegacyDatasource
```

Comment:
Is there a linter or some sort of established way to determine if an import is only being brought in for purposes of type checking? I handrolled something but I'm trying to get away from writing up my own implementations of everything 😆

Reply:
It wouldn't surprise me if this existed but I don't know one.